plugins/Marketplace/Controller.php

```diff
@@ -156,41 +156,6 @@ private function getPrettyLongDate($date)
         return Date::factory($date)->getLocalized(Date::DATE_FORMAT_LONG);
     }
 
-    public function pluginDetails()
```

Comment:
Yeah 🤯

I guess we can keep the method for now for places outside of Marketplace.

Reply:
But that also means we need to keep the Twig template and update it to make use of the `CTAContainer` vue component (and hopefully only that one) :(

Reply:
That feels like a massive PITA, but I can try as the last thing. I would rather revisit and remove links to the plugin detail modal from other places.

Reply:
It will definitely be painful... as in: if a trial is started, we display a notification, but we cannot reload marketplace because marketplace is not available? :exploding_head:

Maybe we can rework it to have a link to marketplace (and perhaps open the modal there directly) instead of the CTA?

Reply:
How about the modal not offering to start a trial for places outside of Marketplace? Basically keep the current behaviour for outside of marketplace, until we figure out how to do that, maybe during the redesign phase?

Reply:
That might be the best way to get around this issue temporarily :thinking:

Reply:
As discussed internally new issue will be lodged for this. Until resolved then we're keeping the method and the template.